M/M4/build_tarballs.jl

```diff
@@ -0,0 +1,29 @@
+using BinaryBuilder
+
+name = "M4"
+version = v"1.4.19"
+
+sources = [
+    ArchiveSource("https://ftp.gnu.org/gnu/m4/m4-$(version).tar.xz",
+                  "63aede5c6d33b6d9b13511cd0be2cac046f2e70fd0a07aa9573a04a82783af96"),
+]
+
+script = raw"""
+cd $WORKSPACE/srcdir/m4-*/
+
+./configure --prefix=${prefix} --host=${target}
+make -j${nprocs}
+make install
+"""
+
+# Windows platforms fail with messages about `__memcpy_chk`
```

Comment:
If you really care about Windows, I think you need to link to `libssp` for `__memcpy_chk`

Reply:
I don't care about Windows (this is for host tools in BB2) but looks like `-fstack-protector` is what is required to get this to work; thanks for the tip!